.travis.yml

```diff
@@ -87,7 +87,7 @@ script:
 # Travis aborts test run if nothing is printed back to STDOUT for some time.
 # -x is used to avoid that.
 - if [ $RUN_FRONTEND_TESTS == 'true' ]; then bash -x scripts/run_frontend_tests.sh --run-minified-tests=true; fi
-- if [ $RUN_PERFORMANCE_TESTS == 'true' ]; then bash scripts/run_performance_tests.sh; fi
+- if [ $RUN_PERFORMANCE_TESTS == 'true' ]; then bash scripts/run_performance_tests.sh --test_target=core.tests.performance_tests.splash_test; fi
```

Comment:
Please add a descriptive `TODO(gvishal):` to add all tests, so that we don't forget to do it later.